aviary/models/large_turboprop_freighter/large_turboprop_freighter.csv

```diff
@@ -0,0 +1,249 @@
+aircraft:nacelle:fineness, 0.38, unitless
+aircraft:nacelle:form_factor, -1, unitless
+aircraft:nacelle:mass_specific, 3, lbm/ft**2
+aircraft:strut:area_ratio, 0, unitless
+aircraft:strut:attachment_location, 0, ft
+aircraft:strut:attachment_location_dimensionless, 0, unitless
+aircraft:strut:dimensional_location_specified, False, unitless
+aircraft:strut:fuselage_interference_factor, 0, unitless
+aircraft:strut:mass_coefficient, 0, unitless
+aircraft:vertical_tail:area, 0, ft**2
+aircraft:vertical_tail:aspect_ratio, 1.81, unitless
+aircraft:vertical_tail:form_factor, -1, unitless
+aircraft:vertical_tail:mass_coefficient, 0.2035, unitless
+aircraft:vertical_tail:moment_ratio, 2.809, unitless
+aircraft:vertical_tail:sweep, 0, deg
+aircraft:vertical_tail:taper_ratio, 0.296, unitless
+aircraft:vertical_tail:thickness_to_chord, 0.15, unitless
+aircraft:vertical_tail:volume_coefficient, 0.05355, unitless
+aircraft:wing:aspect_ratio, 10.078, unitless
+aircraft:wing:center_distance, 0.45, unitless
+aircraft:wing:choose_fold_location, True, unitless
+aircraft:wing:flap_chord_ratio, 0.28, unitless
+aircraft:wing:flap_deflection_landing, 15, deg
+aircraft:wing:flap_deflection_takeoff, 10, deg
+aircraft:wing:flap_drag_increment_optimum, 0.23, unitless
+aircraft:wing:flap_lift_increment_optimum, 1.4, unitless
+aircraft:wing:flap_span_ratio, 0.6, unitless
+aircraft:wing:flap_type, double_slotted, unitless
+aircraft:wing:fold_dimensional_location_specified, False, unitless
+aircraft:wing:fold_mass_coefficient, 0, unitless
+aircraft:wing:folded_span, 0, ft
+aircraft:wing:form_factor, -1, unitless
+aircraft:wing:fuselage_interference_factor, 1, unitless
+aircraft:wing:has_fold, False, unitless
+aircraft:wing:has_strut, False, unitless
+aircraft:wing:height, 14.5, ft
+aircraft:wing:high_lift_mass_coefficient, 1.2, unitless
+aircraft:wing:incidence, 2, deg
+aircraft:wing:loading, 88.846, lbf/ft**2
+aircraft:wing:mass_coefficient, 131.32, unitless
+aircraft:wing:max_lift_ref, 1.5, unitless
+aircraft:wing:max_slat_deflection_landing, 0, deg
+aircraft:wing:max_slat_deflection_takeoff, 0, deg
+aircraft:wing:max_thickness_location, 0.35, unitless
+aircraft:wing:min_pressure_location, 0.3, unitless
+aircraft:wing:mounting_type, 1, unitless
+aircraft:wing:num_flap_segments, 2, unitless
+aircraft:wing:optimum_flap_deflection, 55, deg
+aircraft:wing:optimum_slat_deflection, 45, deg
+aircraft:wing:slat_chord_ratio, 0, unitless
+aircraft:wing:slat_lift_increment_optimum, 0.93, unitless
+aircraft:wing:slat_span_ratio, 0.9, unitless
+aircraft:wing:surface_ctrl_mass_coefficient, 0.588, unitless
+aircraft:wing:sweep, 0, deg
+aircraft:wing:taper_ratio, 0.52, unitless
+aircraft:wing:thickness_to_chord_root, 0.18, unitless
+aircraft:wing:thickness_to_chord_tip, 0.12, unitless
+aircraft:wing:zero_lift_angle, -1, deg
+mission:design:cruise_altitude, 21000, ft
+mission:design:gross_mass, 155000, lbm
+mission:design:mach, 0.475, unitless
+mission:design:range, 0, NM
```

Comment:
with design range = 0 this mission isn't going to fly anywhere

Reply:
No worries, this isn't the final model, just the raw converted GASP file. I'll double check my current version to make sure that got updated

Reply:
in GASP, design range=0 means don't resize the aircraft, just fly the off design

Reply:
Sounds like we have to set the mission_type = Alternate flag in that case and remove the mission.design.range from the csv. because mission.design.range of zero will brick things with the new multi-mission update.